source/dagger/gameplay/team_game/camera.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "core/system.h"
+
+using namespace dagger;
+
+class CameraSystem : public System
```

Comment:
don't forget to use team_game namespace here

Reply:
To put `CameraSystem` in `team_game` namespace?

Reply:
yup, every class/struct from gameplay/team_game